src/main/java/com/axis/jenkins/plugins/eiffel/eiffelbroadcaster/QueueListenerImpl.java

```diff
@@ -45,11 +44,23 @@ of this software and associated documentation files (the "Software"), to deal
  */
 @Extension
 public class QueueListenerImpl extends QueueListener {
+    private static final Logger LOGGER = LoggerFactory.getLogger(QueueListenerImpl.class);
```

Comment:
It's no big deal, but a pet peeve of mine: https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case

Reply:
Fixed in a new prequel commit.